travvy-server/app.js

```diff
@@ -3,18 +3,27 @@ const express = require('express')
 const bodyParser = require('body-parser')
 const cors = require('cors')
 const morgan = require('morgan')
-const { sequelize } = require('./models')
-const config = require('./config/config')
+const { sequelize } = require('./src/models')
+const config = require('./src/config/config')
 
 const app = express()
 app.use(morgan('combined'))
 app.use(bodyParser.json())
 app.use(cors())
 
-require('./routes')(app)
+require('./routes/routes')(app)
+
+//Handle production 
+if(process.env.NODE_ENV == 'production'){
+  //Static folder
+  app.use(express.static(__dirname + '/public/'))
+
+  //Handle single page 
+  app.get(/.*/,(req, res) => res.sendFile(__dirname +'/public/index.html'))
+}
 
 // Setup sequelize server
 sequelize.sync().then(() => {
-  app.listen(process.env.PORT || 8081)
+  app.listen(process.env.PORT || 5000)
```

Comment:
are we not supposed to use port 8081? I thought that was the back end server port

Reply:
There were some issues with deployment because sequelize was on the same server so I changed the port which seemed to have fixed the issue